rq_scheduler/scheduler.py

```diff
@@ -152,6 +153,24 @@ def schedule(self, scheduled_time, func, args=None, kwargs=None, interval=None,
                               job.id)
         return job
 
+
+    def cron(self, crontab_string, func, args=None, kwargs=None, queue_name=None):
+        """
+        Schedule a cronjob
+        """
+        job = self._create_job(func, args=args, kwargs=kwargs, commit=False,
```

Comment:
We should validate that `crontab_string` is in a valid format before creating this job in Redis.